src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1704,6 +1704,7 @@
     matching Id that are integers unique for the containing method body.
     -->
     <Field Name="Id" Type="int"/>
+    <Field Name="ForceCopyOfNullableValueType" Type="bool"/>
```

Comment:
:memo: Note to future self: For conditional access we do want a copy when dealing with an unconstrained type, for the nullable value type scenario to behave properly. 
But we also use this bound node in a few other scenarios (string concat and fixed statement) that need different semantics (we shouldn't get any copy).